### ISSUE TITLE
Plain-text console output feature

### ISSUE THREAD

**Issue by Hawk777** [jenkinsci/pipeline-graph-view-plugin#301]
### What feature do you want to see added?

TLDR: For large console output, a link to a `text/plain` version of the step’s whole output.

When a step’s console output is tens of megabytes, it can be awkward to navigate embedded in a small section of a page; for that matter, in HTML form, in at least some browsers, console output of that magnitude takes quite a long time to load and is slow to navigate and search. Blue Ocean has a Show Complete Log button in such cases, which links to a URL which returns a `text/plain` version of the step’s complete console output, which, being shown in the full browser window and being much lighter-weight, is easier to navigate. For even larger console output, or more complex searching needs  (e.g. regex), it’s also the perfect URL to download to file and open in a text editor. For my team’s needs, I think this is the only thing missing from Pipeline Graph View compared to Blue Ocean, other than perhaps a bugfix or two which are reported here.
